nyxtools/vector.py

```diff
@@ -227,8 +231,12 @@ def run(
         daq_duration = yield from bps.rd(self.data_acq_duration)
 
         estimated_total_time_ms = 2*time_to_speed + buffer_time + 2*shutter_time + daq_duration
-        timeout = 5*estimated_total_time_ms/1000.0
+        self.timeout = 5*estimated_total_time_ms/1000.0
+        self.ready = True
 
+    def move(self):
+        if not self.ready:
```

Comment:
we will be reusing the vector object multiple times so would it make sense to reset the `self.ready` flag back to false after the move is started?

this will ensure the flag will be false if another `move()` is attempted before another `prepare_move(params)` call

Reply:
No reason to prepare_move each time, at least not at this level. When using the flyer, such a feature should be added.

Tested at NYX, we can move multiple times without issue.